### PR TITLE
fix: add john babu's profile

### PR DESCRIPTION
**Pull request by johnbabu021**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/897"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add John Babu`

File: public/data/john.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "johnbabu",
```
Comment (syedsohan) [inline public/data/john.json:2]:
```suggestion
  "name": "John Babu",
```

File: public/data/john.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "type": "personal",
+  "bio": "open source enthusiast",
```
Comment (syedsohan) [inline public/data/john.json:4]:
```suggestion
  "bio": "Open source enthusiast",
```

File: public/data/john.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/john86208561",
+      "icon": "twitter"
+    }
+  ],
+  "milestone": [
+    {
+      "title": "joined gitHub",
```
Comment (syedsohan) [inline public/data/john.json:20]:
```suggestion
      "title": "Joined GitHub",
```

File: public/data/john.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      "data": "December 2019",
+      "color": "grey",
+      "description": "started coding"
```
Comment (syedsohan) [inline public/data/john.json:23]:
```suggestion
      "description": "Started coding"
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by Vyvy-vi — APPROVED** [review]
Looks good to me!

**Pull request merged**